Neos.Ui/shortcut-handler/src/index.tsx

```diff
@@ -0,0 +1,120 @@
+import {INeosContextProperties} from "@sitegeist/archaeopteryx-neos-bridge";
+import {IEditor} from "@sitegeist/archaeopteryx-core";
+import {SynchronousRegistry} from "@neos-project/neos-ui-extensibility";
+import {ILinkOptions} from "@sitegeist/archaeopteryx-core/lib/domain";
+
+export const executeCommand = (editor: NeosEditor, command: string, argument: any, reFocusEditor = true) => {
+    editor.execute(command, argument);
+    if (reFocusEditor) {
+        editor.editing.view.focus();
+    }
+};
+export function registerShortcutHandler (
+    neosContextProperties: INeosContextProperties,
+    editor: IEditor
+): void {
+    const {globalRegistry, store} = neosContextProperties;
+    const ckeditor5Registry = globalRegistry.get('ckEditor5');
+    if (!ckeditor5Registry) {
+        console.warn('[Sitegeist.Archaeopteryx]: Could not find ckeditor5 registry.');
+        console.warn('[Sitegeist.Archaeopteryx]: Skipping registration of keyboard shortcut...');
+        return;
+    }
+
+    const ckeditor5Configuration = ckeditor5Registry.get<SynchronousRegistry<any>>('config')
+    if (!ckeditor5Registry) {
+        console.warn('[Sitegeist.Archaeopteryx]: Could not find ckeditor5 config registry.');
+        console.warn('[Sitegeist.Archaeopteryx]: Skipping registration of keyboard shortcut...');
+        return;
+    }
+
+    ckeditor5Configuration.set('keystrokes', (ckEditorConfiguration: { plugins: ((ckEditorInstance: NeosEditor) => void)[]; }) => {
+        ckEditorConfiguration.plugins.push((ckEditorInstance) => {
+            ckEditorInstance.keystrokes.set('Ctrl+k', async (_: any, cancel: () => void) => {
```

Comment:
Is it easily possible to make the shortcut configurable in a configuration file?